src/content/docs/docs/support/quick-start.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Quick start guide
+tableOfContents: false
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import { LinkOut } from "@interledger/docs-design-system";
+
+<Steps>
+1. Download and install the Interledger Foundation's Web Monetization extension. 
+    * <LinkOut href="https://chromewebstore.google.com/detail/web-monetization-extensio/oiabcfomehhigdepbbclppomkhlknpii">Google Chrome and Chromium-based browsers</LinkOut>
+    * <LinkOut href="https://addons.mozilla.org/en-US/firefox/addon/web-monetization-extension/">Mozilla Firefox</LinkOut>
+    * <LinkOut href="https://microsoftedge.microsoft.com/addons/detail/web-monetization-extensio/imjgemgmeoioefpmfefmffbboogighjl">Microsoft Edge</LinkOut>
```

Comment:
Maybe we should move these links into some shared constants or component?

Reply:
We can do this in the new version of the web monetization docs (which go beyond the extension docs)